myosuite/envs/myo/myochallenge/bimanual_v0.py

```diff
@@ -236,7 +237,8 @@ def get_reward_dict(self, obs_dict):
                 ("pass_err", pass_dist + np.log(pass_dist + 1e-3)),
                 # Must keys
                 ("sparse", 0),
-                ("solved", self.check_solve(goal_dis)),
+                ("goal_dist", goal_dis), 
+                ("solved", goal_dis < 0.05),
```

Comment:
can we have this as a variable?

Reply:
Updated in this commit: https://github.com/MyoHub/myosuite/pull/213/commits/3281da5c665fcb3a114ad7f9822c67399f677acf